File: csharp-dotnet2/src/main/CsharpDotNet2/estuary-client/Model/ApiStorageProviderResp.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class ApiStorageProviderResp {
    /// <summary>
    /// Gets or Sets Addr
    /// </summary>
    [DataMember(Name="addr", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "addr")]
    public AddressAddress Addr { get; set; }

    /// <summary>
    /// Gets or Sets Name
    /// </summary>
    [DataMember(Name="name", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "name")]
    public string Name { get; set; }

    /// <summary>
    /// Gets or Sets Suspended
    /// </summary>
    [DataMember(Name="suspended", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "suspended")]
    public bool? Suspended { get; set; }

    /// <summary>
    /// Gets or Sets SuspendedReason
    /// </summary>
    [DataMember(Name="suspendedReason", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "suspendedReason")]
    public string SuspendedReason { get; set; }

    /// <summary>
    /// Gets or Sets Version
    /// </summary>
    [DataMember(Name="version", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "version")]
    public string Version { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class ApiStorageProviderResp {\n");
      sb.Append("  Addr: ").Append(Addr).Append("\n");
      sb.Append("  Name: ").Append(Name).Append("\n");
      sb.Append("  Suspended: ").Append(Suspended).Append("\n");
      sb.Append("  SuspendedReason: ").Append(SuspendedReason).Append("\n");
      sb.Append("  Version: ").Append(Version).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: dart/lib/model/api_storage_provider_resp.dart
part of swagger.api;

class ApiStorageProviderResp {
  
  AddressAddress addr = null;

  String name = null;

  bool suspended = null;

  String suspendedReason = null;

  String version = null;

  ApiStorageProviderResp();

  @override
  String toString() {
    return 'ApiStorageProviderResp[addr=$addr, name=$name, suspended=$suspended, suspendedReason=$suspendedReason, version=$version, ]';
  }

  ApiStorageProviderResp.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    addr = new AddressAddress.fromJson(json['addr']);
    name = json['name'];
    suspended = json['suspended'];
    suspendedReason = json['suspendedReason'];
    version = json['version'];
  }

  Map<String, dynamic> toJson() {
    return {
      'addr': addr,
      'name': name,
      'suspended': suspended,
      'suspendedReason': suspendedReason,
      'version': version
     };
  }

  static List<ApiStorageProviderResp> listFromJson(List<dynamic> json) {
    return json == null ? new List<ApiStorageProviderResp>() : json.map((value) => new ApiStorageProviderResp.fromJson(value)).toList();
  }

  static Map<String, ApiStorageProviderResp> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, ApiStorageProviderResp>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new ApiStorageProviderResp.fromJson(value));
    }
    return map;
  }
}
